docs/index.md

```diff
@@ -41,21 +41,28 @@ The video player supports the following video formats: `MP4`, `WebM`, `HLS`, `DA
 import Playable from 'playable';
 
 document.addEventListener('DOMContentLoaded', function() {
-  const player = Playable.create({
+  const config = {
     size: {
       width: 700,
       height: 394
     },
     src: 'http://my-url/video.mp4',
     preload: 'metadata'
-  });
+  };
+  const theme = {
+    progressColor: "#AAA"
+  }
+
+  const player = Playable.create(config, theme);
```

Comment:
I think the first example in `index.md` should be minimal code to run player.

Mb remove theme config here and change text "You can pass your own [theme](/themes)" with smth like "You can change player theme with your config" + examle

Reply:
agree